unlock-app/src/__tests__/middlewares/providerMiddleware.test.ts

```diff
@@ -69,19 +68,19 @@ describe('provider middleware', () => {
       providerMiddleware(config)({ getState, dispatch })(next)(erroneousAction)
     })
 
-    it('should set an error and return if the call to enable fails', () => {
-      expect.assertions(3)
-
+    it('should set an error and return if the call to enable fails', done => {
+      expect.assertions(2)
       config.providers['UNLOCK'].enable = jest.fn(() => {
-        throw new Error('The front fell off.')
+        // eslint-disable-next-line promise/param-names
```

Comment:
This is here because if I don't do this, both TypeScript and eslint will complain about resolve being unused.